app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectCareTeam.php

```diff
@@ -112,7 +112,7 @@ public function table(Table $table): Table
                             ->label('User')
                             ->searchable()
                             ->required()
-                            ->options(User::query()->tap(new HasLicense(Prospect::getLicenseType()))->pluck('name', 'id')),
+                            ->options(User::query()->tap(new HasLicense(Prospect::getLicenseType()))->whereDoesntHave('prospectCareTeams')->pluck('name', 'id')),
```

Comment:
Users can be a part of multiple care teams. So this should be changed to specifically filter out Users that are already in the Educatables specific care team.